codegen/masm/src/stackify/emit/binary.rs

```diff
@@ -490,6 +534,7 @@ impl<'a> OpEmitter<'a> {
                 self.emit(Op::Div);
             }
             Type::U32 | Type::U16 | Type::U8 => self.unchecked_div_u32(),
+            Type::I32 => self.checked_div_i32(),
```

Comment:
Is it temporary until we have an unchecked division for i32 implemented? OTOH, `intrinsics::i32::checked_div` does not check for an overflow. I'm confused.